src/components/Footer/Fotter.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { Todo } from '../../types/Todo';
+import { TodoStatus } from '../../types/Status';
+
+type Props = {
+  todos: Todo[];
+  status: TodoStatus;
+  onStatusChange: (arg: TodoStatus) => void;
```

Comment:
```suggestion
  onStatusChange: (status: TodoStatus) => void;
```
`arg` is a bad name, even in types